include/deal.II/matrix_free/vector_access_internal.h

```diff
@@ -340,6 +346,11 @@ namespace internal
                        VectorizedArrayType &res,
                        std::integral_constant<bool, true>) const
     {
+#ifdef DEBUG
+      for (unsigned int v = 0; v < VectorizedArrayType::size(); ++v)
+        vector_access(vec, indices[v] + constant_offset);
+#endif
```

Comment:
Same here, I'd prefer an `#if`/`#else` statement to avoid confusion about doing the same thing twice.